### PR TITLE
提供配置自定义 model 路径

### PR DESCRIPTION
**Pull request by JZLeung**
可以让一个项目同时配置多个数据库，如 mongodb 和 mysql.

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by thonatos** [comment]
@huacnlee 

没问题的话 `merge` 了吧

File: lib/loader.js [hunk 1]
```diff
@@ -69,8 +69,8 @@ function* authenticate(app) {
   }
 }
 
-function loadModel(app) {
-  const modelDir = path.join(app.baseDir, 'app/model');
+function loadModel(app, config) {
+  const modelDir = path.join(app.baseDir, config.dir || 'app/model');
```
Comment (huacnlee) [inline lib/loader.js]: `config.dir` -> `config.baseDir`

Comment (BaffinLee) [inline lib/loader.js]: 1. 默认值放 `defaultConfig` 里会不会好一点？
2. 文档需要更新

Comment (thonatos) [reply lib/loader.js]: 嗯，放 `defaultConfig` 确实更好

**Comment by fengmk2** [comment]
impl in https://github.com/eggjs/egg-sequelize/pull/60

**Pull request closed without merging**